extensions/EXT/SPV_EXT_mesh_shader.asciidoc

```diff
@@ -158,6 +159,19 @@ Modifications to the SPIR-V Specification
 Add *OpEmitMeshTasksEXT* to the list of Termination Instructions.
 
 (Modify Section 2.16.1, Universal Validation Rules) ::
++
+Modify the list following the statement:
++
+====
+It is invalid for a pointer to be an operand to any instruction other than:
+====
++
+to include:
++
+====
+* <<OpEmitMeshTasksEXT,*OpEmitMeshTasksEXT*>>
+====
++
```

Comment:
```suggestion
+
Add the following to the end of the section:
```

the rendered html looks ok (doesn't fully looks like one list) but helps to have a clear separation